### PR TITLE
Clean-up UI kitten

### PR DESCRIPTION
**Pull request by ishrak-imam**
### Description

Clean-up ui-kitten from the project and minor refactor where needed.

### CODE REVIEW

**Comment by geastwood** [comment]
With this PR, are we completely taking out UI kitten?

**Comment by ishrak-imam** [comment]
> With this PR, are we completely taking out UI kitten?

Yes.

**Comment by geastwood** [comment]
Judging from `package.json`, we take out UI kitten completely. And also replace momentjs with dayjs?

**Comment by ishrak-imam** [comment]
> Judging from `package.json`, we take out UI kitten completely. And also replace momentjs with dayjs?

Yes, usage of moment was not so widespread in the code. I replaced it with dayjs since moment is not actively developed anymore.

**Comment by geastwood** [comment]
This does require a click through the app. IMO

**Comment by ishrak-imam** [comment]
> This does require a click through the app. IMO

I am doing it. But more eyes are always better 👁️

File: src/identityUtils.ts [hunk 1]
```diff
@@ -2,7 +2,7 @@ import {IdentityJudgement} from '@polkadot/types/interfaces';
```
Comment (silva-fj) [inline src/identityUtils.ts:2]: can you move this file to `src/utils/identity.ts`?

File: src/ui/components/WarningDialog.tsx [hunk 2]
```diff
@@ -14,7 +15,7 @@ function WarningDialog(props: PropTypes) {
   return (
     <Layout style={globalStyles.centeredContainer}>
       <Layout style={styles.textContainer}>
-        <Icon style={styles.icon} fill={'#ffcc00'} name="alert-circle-outline" />
+        <Icon size={20} color={'#ffcc00'} name="alert-circle-outline" />
```
Comment (silva-fj) [inline src/ui/components/WarningDialog.tsx]: let's add the color to theme as **warning**

File: src/ui/screens/Council/CandidateScreen.tsx [hunk 3]
```diff
@@ -20,13 +20,25 @@ type ScreenProps = {
...
+}
+
 export function CandidateScreen({route, navigation}: ScreenProps) {
   const accountId = route.params.accountId;
   const backing = route.params.backing;
   const screenTitle = route.params.title;
   const {data: councilData, isLoading: isLoadingCouncil} = useCouncil();
   const {data: identityInfoData, isLoading: isLoadingIdentityInfo} = useAccountIdentityInfo(accountId);
-  const theme = useTheme();
+  // const theme = useTheme();
```
Comment (silva-fj) [inline src/ui/screens/Council/CandidateScreen.tsx]: remove?

File: src/ui/screens/Council/MotionsScreen.tsx [hunk 4]
```diff
@@ -102,57 +100,48 @@ function Motion({item}: {item: DeriveCollectiveProposal}) {
...
+                      </Button>
                     );
                   } else if (isVoteable) {
                     return (
-                      <View style={motionStyle.buttons}>
-                        <Button status={'danger'} size={'tiny'} onPress={onPressNay}>
-                          Nay
+                      <View style={globalStyles.rowAlignCenter}>
+                        <Button onPress={onPressNay} color={colors.error} mode="outlined">
+                          Nye
```
Comment (silva-fj) [inline src/ui/screens/Council/MotionsScreen.tsx]:
```suggestion
                          Nay
```

File: android/app/build.gradle [hunk 5]
```diff
@@ -83,10 +83,7 @@ project.ext.react = [
     enableHermes: false,
 ]
 
-project.ext.vectoricons = [
-    // TODO: remove Ionicons after updating completely to rnpaper and removing ui-kitten
-    iconFontNames: [ 'Ionicons.ttf', 'MaterialIcons.ttf', 'MaterialCommunityIcons.ttf'] // Name of the font files you want to copy
-]
+project.ext.vectoricons = []
```
Comment (silva-fj) [inline android/app/build.gradle]: If I remember correctly if you don't specify the fonts it will copy all the fonts that are bundled in the library

**Review by silva-fj — COMMENTED** [review]
Good job, let's make sure to do a round of general testing after this is merged.

**Review by silva-fj — APPROVED** [review]

**Pull request merged**